ctsit/metab_import
# Change pub ingest to use harvard catalyst profiles disambiguation

**What does this change do?** _Please be clear and concise._

Change the pub ingest to use the disambiguation service when there are known
publications in the supplemental database.

**Why was this change made?** _Including an issue number is sufficient. Otherwise, briefly explain the benefit of the change._

To reduce number of non-matches of publications for authors.

## Verification

_List the steps needed to make sure this thing works. Be precise._

1.  Insert some data into the Publications table in the supplemental database.
ex: `INSERT INTO publications VALUES ('11523076',709,TRUE),('32004512', 709, FALSE)`
2. Run the pub_ingest tool.

## Affirmations

All of these should have a check by them. Any exception requires an explanation.
